test-requirements.txt

```diff
@@ -4,3 +4,4 @@ responses
 # Python Dependencies
 PyYAML
 xmlunittest
+odoo14-addon-product_import @ git+https://github.com/OCA/edi@refs/pull/687/head#subdirectory=setup/product_import
```

Comment:
Please, always keep this change in a separated commit like `Add TMP test dependency`.
Makes it prominent and, when done, we can simply trash the commit.